app/src/main/java/com/javahelp/frontend/ProviderRegistrationViewmodel.java

```diff
@@ -0,0 +1,17 @@
+package com.javahelp.frontend;
+
+import android.app.Application;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+public class ProviderRegistrationViewmodel extends AndroidViewModel {
+
+
```

Comment:
Here I'm also thinking of a case where a constructor takes in `null` application as well. Is it needed?